src/popup/components/InputField.vue

```diff
@@ -237,6 +240,14 @@ export default defineComponent({
       },
     );
 
+    onMounted(() => {
```

Comment:
This is weirdly the same changes as here. https://github.com/superhero-com/superhero-wallet/pull/3291/files#diff-d176ac880b039692a793823c27562dd25951237c941c78490de5fc64e8eb6f82R243